CONTRIBUTING.md

```diff
@@ -1,23 +1,83 @@
 Contributing to the openPMD standard
 ======================================
 
-Participation
--------------
+The openPMD standard can evolve in order to accommodate the needs of the community. This results in successive [*versions* of the standard](https://github.com/openPMD/openPMD-standard/blob/1.0.1/STANDARD.md#the-versions-of-this-standard). This document explains the process by which the standard evolves, and how to contribute to it.
+
+Update Cycle
+------------
 
-We participate via *pull requests* on GitHub.
-Improvements will be discussed and adressed in those.
+The updates of the openPMD standard go through a 3-step process:
 
-Future proposals and strategies can be discussed in GitHub *issues*.
+- **Issues:** Anyone can propose changes to the standard, request new features,
+ask for clarification, etc. by opening a new issue
+[here](https://github.com/openPMD/openPMD-standard/issues). The proposed
+changes/features/question will then be discussed by the members of the
+openPMD community (or anyone interested, really) in the `Comments` section of
+this issue. The openPMD admins will usually add a label specifying the version
```

Comment:
maybe replace admins with maintainers

Reply:
we add labels for:
- scope and
- impact of the change

The versions that will potentially implement a change can be the next matching version we work on or any later, depending on the process of the issue and when consensus is reached. (Documented in "projects").

For some issues we add also a "milestone" which is purely informative, e.g. to document a prior release a question refers to.